cpu/from-lscpu-arm.py

```diff
@@ -0,0 +1,74 @@
+#! /usr/bin/env python
```

Comment:
Could we add something to the README about how to use this and when to update the ARM cpu definition ?